packages/fluentui/react-northstar-prototypes/src/prototypes/instructionMessage/instructionMessage.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useFluentContext } from '@fluentui/react-bindings';
+import { Box, Ref } from '@fluentui/react-northstar';
+
+interface InstructionMessageProps {
+  message: string;
+  role?: string;
+  'aria-label'?: string;
+}
+
+const InstructionMessage: React.FunctionComponent<InstructionMessageProps> = props => {
+  const { children, message, ...rest } = props;
+
+  const narrationHappened = React.useRef(false);
+  const instructionRef = React.useRef<HTMLElement>();
+  const timeoutMessageNarrate = React.useRef<number>();
+  const timeoutMessageRemove = React.useRef<number>();
+  const context = useFluentContext();
+  const areaFocusableElements = () =>
+    instructionRef?.current?.querySelectorAll('a, button, input, textarea, select, details, [tabindex]');
+
+  const narrate = (instructionMessage, priority = 'polite') => {
+    const element = document.createElement('div');
+    element.setAttribute(
+      'style',
+      'position: absolute; left: -10000px; top: auto; width: 1px; height: 1px; overflow: hidden;',
+    );
+    element.setAttribute('aria-live', priority);
+    document.body.appendChild(element);
```

Comment:
```suggestion
    context.target.<SOMETING-SOMETHING>.body.appendChild(element);
```
Shouldn't it also be context depended?